to-do-list.js

```diff
@@ -0,0 +1,8 @@
+function createList(event) {
+  event.preventDefault();
+  let input = document.querySelector('#text-field').value;
+  let newListItem = document.createElement('li');
+  let newContent = document.createTextNode(input);
```

Comment:
Prefer const over let

Reply:
Ok. This part was commented out and not used. I replaced let with const however, I am not quite sure if I understand the difference.  Should we use const so the value of variables will not be reassigned?